src/components/ProfileImage/ProfileImage.module.scss

```
@@ -0,0 +1,32 @@
+.profile-image-wrapper {
+  @include flex-center;
+
+  position: relative;
+  width: max-content;
+}
+
+.profile-image {
+  overflow: hidden;
+  border-radius: 999rem;
```

```suggestion
  border-radius: 100%;
```
요런 방법도 있을거 같아용~! 밑에는 99고 여기는 999라서 통일하면 좋을거 같아요?

---

오오!!!! 그런 방법이 첨 알았슴둥~!